manifests/init.pp

```diff
@@ -30,32 +40,53 @@
 #   *If* the $servers array above starts with '127.0.0.1' or '::1', then
 #   the system will set itself up as a caching nameserver unless this is set
 #   to false.
-#
+# @param sortlist
+#   Optional Array of address/netmask pairs that allow addresses returned by
+#   gethostbyname to be sorted.
+# @param extra_options
+#   Optional Array to put any options that may not be covered by the variables
+#   below. These will be appended to the options string.
 class resolv (
   Simplib::Netlist           $servers        = simplib::lookup('simp_options::dns::servers', { 'default_value' => ['127.0.0.1'] }),
   Simplib::Netlist           $search         = simplib::lookup('simp_options::dns::search', { 'default_value'  => [$facts['domain']] }),
```

Comment:
Same problem as above:  The type of $search is incorrect here, but fixed in the EPP's $search parameter type.   I think this should be fixed in both places and the CHANGELOG should say that the apparent API change will have no impact on valid resolv parameters used in the older resolv module.  In other words, if you got it to work on the system, it will be valid with the new module.

Reply:
Same reply as above: The CHANGELOG entry is a good idea.